datalad/support/gitrepo.py

```diff
@@ -1324,12 +1324,9 @@ def get_toppath(cls, path, follow_up=True, git_options=None):
             cmd.extend(git_options)
         cmd += ["rev-parse", "--show-toplevel"]
         try:
-            toppath, err = GitRunner().run(
-                cmd,
-                cwd=path,
-                log_stdout=True, log_stderr=True,
-                expect_fail=True, expect_stderr=True)
-            toppath = toppath.rstrip('\n\r')
+            out = GitWitlessRunner(cwd=path).run(
```

Comment:
a note: coverage marked this as not-covered for me -- doesn't make sense since it would be the first line in `try:` and prior lines are covered.